src/Modal.js

```diff
@@ -182,6 +184,31 @@ class Modal extends React.Component {
     }
   }
 
+  handleTab(e) {
+    if (e.which !== 9) return;
+
+    const focusableChildren = this._element.querySelectorAll(focusableElements.join(', '));
+    const currentFocus = document.activeElement;
```

Comment:
`document.activeElement` can have issues in IE, specially within iframes.
I just ran into it the other day, https://github.com/Availity/react-block-ui/issues/18

Reply:
Wrapped this in a try/catch, defaulting to the first focusable child. Tested on IE10, IE11 on win7 and MSEDGE on win10. Let me know if I missed a version.